src/LaravelRequestDocs.php

```diff
@@ -325,31 +361,29 @@ public function lrdDocComment(string $docComment): string
 
     public function appendExample(Doc $doc): void
     {
-            try {
-                $controllerName = class_basename($doc->getController());
-                $modelName = Str::replace('APIController', '', $controllerName);
-                $fullModelName = "App\\Models\\" . $modelName;
-
-                if (!class_exists($fullModelName)) {
-                    return;
-                }
+        try {
+            $controllerName = class_basename($doc->getController());
+            $modelName      = Str::replace('APIController', '', $controllerName);
+            $fullModelName  = "App\\Models\\" . $modelName;
```

Comment:
In this case, what I could suggest would be two new keys in the configuration, one to inform the path of the application's models (it would be a way for projects that use a different directory for the models to be able to customize the component). And for controllers we could add a configuration to add a regex to extract the name of the models through the name of the controller. I believe that this way we could customize this extraction both the model directory and the controller suffix. Remembering that these points I went through were thinking about the convention adopted by Laravel, so unfortunately if the user chooses to create a structure that is very distant from the current structure, it will not be possible for us to use some tools. But if there is another option on how to get the model's path through the route/request/controller, I can make the modification

Reply:
I made an implementation where the pattern must be passed in the settings to extract the model name from the controller name and the factory directory must also be entered so the values ​​are not fixed and can be easily configured by the user